### PR TITLE
add missing std::vector

### PR DESCRIPTION
**Pull request by neheb**
small size reduction. fmt seems to like vector.

This also avoids using std::initializer_list, which should not ve used.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

File: src/database/sql_database.cc [hunk 1]
```diff
@@ -2515,18 +2515,18 @@ void SQLDatabase::migrateMetadata(int objectId, const std::string& metadataStr)
...
+                identifier("property_name"),
+                identifier("property_value"),
+            };
             auto values = std::vector {
                 fmt::format("{}", objectId),
                 fmt::format("{}", key),
                 fmt::format("{}", val),
             };
-            insert(METADATA_TABLE, fields, values);
+            insert(METADATA_TABLE, std::move(fields), std::move(values));
```
Comment (KarlStraussberger) [inline src/database/sql_database.cc:2529]: Move does not make sense to me if the target is a const ref. Recreating the vector in each loop seems expensive, too.

Comment (KarlStraussberger) [inline src/database/sql_database.cc:2529]: Move does not make sense to me if the target is a const ref. Recreating the vector in each loop seems expensive, too.

**Comment by neheb** [comment]
How about now?

**Comment by KarlStraussberger** [comment]
Great!

**Pull request merged**